jobs/upload-dashboards-objects/templates/bin/generate-tenants

```diff
@@ -0,0 +1,10 @@
+set -eu
+
+<%
+  api = p("cloudfoundry.system_domain")
+  client = p("cloudfoundry.client_id")
+  password = p("cloudfoundry.client_password")
+%>
+cf api "<%= api %>"
+cf auth "<%= client %>" "<%= password %>" --client-credentials
+tenant_list=$(cf orgs| tail -n +4 >> org-tenants.txt)
```

Comment:
```suggestion
cf orgs| tail -n +4 > "$STORE_DIR/org-tenants.txt"
```

No need to use a subshell since we're not doing anything with `tenant_list`. And using a subshell could hide the exit code.

Also I think we want to use `>` instead of `>>` so that `org-tenants.txt` is overwritten every time

Last thing: can we write the file to `/var/vcap/store/upload-dashboards-objects/orgs-tenant`. It occurs to me that we probably shouldn't write temporary files in the actual job directory

Reply:
sure